src/main/java/nl/hsac/fitnesse/fixture/slim/web/BrowserTest.java

```diff
@@ -935,6 +935,23 @@ public String valueForIn(String place, String container) {
         return valueFor(element);
     }
 
+    @WaitUntil(TimeoutPolicy.RETURN_NULL)
+    public String valueOfAttributeOn(String attribute, String place) {
+        return valueOfAttributeOnIn(attribute, place, null);
+    }
+
+    @WaitUntil(TimeoutPolicy.RETURN_NULL)
+    public String valueOfAttributeOnIn(String attribute, String place, String container) {
+        String result;
+        WebElement element = getElementToRetrieveValue(place, container);
+        try {
+            result = element.getAttribute(attribute);
+        } catch(NullPointerException e) {
```

Comment:
Let's not catch NullPointers but do a check whether element is null before trying to get the attribute.